build.gradle

```diff
@@ -255,3 +255,24 @@ def deployer = uploadArchives.repositories.mavenDeployer
     }.each { dep -> dep.optional = true }
 }
 
+
+apply plugin: 'findbugs'
+//findbugs
+findbugs {
+    toolVersion = "3.0.1"
+    //report only high severity bugs for now
+    reportLevel = "high"
+    //the code base is pretty small so use max effort
+    effort = "max"
+    //we don't want to run findbugs on the test code yet
+    sourceSets = [sourceSets.main]
+    //exclude a couple of known bugs until we get the chance to fix them
+    excludeFilter = file("findbugs_excludes.xml")
+}
+
+tasks.withType(FindBugs) {
+    reports {
+        xml.enabled = false
```

Comment:
Does Jenkins automatically read a findbugs generated xml report? If it does then yes, it would be good to switch it on. The HTML is much more human readable which is why I was using that one at the moment.